doc/contributing/hw/racl/README.md

```diff
@@ -1,64 +1,92 @@
+If the access is denied then the error is logged into an Error-Log register.
+
+### RACL Policies
+Each policy register takes the form of a pair of permission bitmaps, `ReadPerm` and `WritePerm`, where each bit corresponds to a role.
+E.g., if 16 different roles are defined, the register looks as follows, where the bit position of each role is defined by the role encoding that is specified via the `role_id` in the [configuration file](#top-level-racl-configuration).
+
+|                 | Role 0 | Role 1 | ... | Role 14 | Role 15 |
+|-----------------|:------:|:------:|:---:|:-------:|:-------:|
+| *Bit position:* | *0*    | *1*    | ... | *14*    | *15*    |
+| `WritePerm`     |        |        |     |         |         |
+| `ReadPerm`      |        |        |     |         |         |
+
+Each `ReadPerm` bit indicates if the corresponding role (see column header) has read permission to objects (e.g., registers) controlled by this policy.
+Each `WritePerm` bit indicates if the corresponding role has write permission.
+When the bit is 1, the corresponding permission is granted and when it is 0 it is denied.
 
+The policy registers shall be mapped as shown below.
+Note, the policy registers shall be 64-bit aligned to allow room for future use.
+
+| Name     | Offset | Bits 31:16  | Bits 15:0  |
+|----------|:------:|:-----------:|:----------:|
+| Policy 0 | 0x000  | `writePerm` | `readPerm` |
+|          | 0x004  | reserved    | reserved   |
+| Policy 1 | 0x008  | `writePerm` | `readPerm` |
+|          | 0x00c  | reserved    | reserved   |
+| Policy 2 | 0x010  | `writePerm` | `readPerm` |
+|          | 0x014  | reserved    | reserved   |
+| Policy 3 | 0x018  | `writePerm` | `readPerm` |
+|          | 0x01c  | reserved    | reserved   |
+| Policy 4 | 0x020  | `writePerm` | `readPerm` |
+|          | 0x024  | reserved    | reserved   |
+
+In the [RACL configuration file](#top-level-racl-configuration), exactly one policy must be defined with `rot_private: true`.
+This policy is then used to statically protect the policy registers themselves.
+This means that the read and write permissions are used from this policy as defined in the configuration file at design time.
+For all policy registers, the configuration specified in the configuration file is also the reset default value.
+However, depending on the permissions defined for the `rot_private`-designated policy, the policy registers may also be updated at runtime.
+
+### RACL Controller
+
+The comportable IP [`racl_ctrl` (RACL Controller)](../../../../hw/top_darjeeling/ip_autogen/racl_ctrl/README.md) is the interface between the TileLink bus and the RACL matching logic.
+The RACL Controller [implements the policy registers](../../../../hw/top_darjeeling/ip_autogen/racl_ctrl/doc/registers.md), distributes them to subscribing IPs and collects error logs from them through its [hardware interface](../../../../hw/top_darjeeling/ip_autogen/racl_ctrl/doc/interfaces.md)
```

Comment:
Since this file isn't templated, I think we need to be a bit careful here. I'd suggest something like the following:

```
For a top-level that uses RACL, there is an associated RACL Controller (`racl_ctrl`), which acts as the interface between the TileLink bus and the RACL matching logic.
For example, see the documentation for the [Darjeeling racl_ctrl]( ... ).
```